### ISSUE TITLE
Reduced left menu is not working

### ISSUE THREAD

**Issue by Robin-Fischer-PS** [PrestaShop/PrestaShop#17912]
#### Describe the bug

Reduced left menu is not working in 1.7.7 : When there is sub-items in an item, it should open a sub-menu, but nothing happens.

#### Expected behavior

Explain what you expected to happen instead.

#### Steps to Reproduce

1. Go to BO
2. Reduce left menu
3. Try to open an item with sub-items inside (order for example) : Nothing happens.

**Screenshots**

https://drive.google.com/file/d/1ogI4SqnjeJ6FvC5XE8LE7Xt7YpheSVfs/view?usp=sharing

#### Additional information

* PrestaShop version: 1.7.7
* PHP version: N/A

**Comment by davidglezz** [comment]
This is caused by new Custom Scrollbar that add `.ps` class to `nav#nav-sidebar` and hides overflow (red square) that shows submenu.
```css
.ps {
	overflow: hidden!important; /* Needed for the Custom Scrollbar */
}

.page-sidebar-closed:not(.mobile) .nav-bar {
    overflow: visible; /* Needed for the submenu */
    ...
}
```

![image](https://user-images.githubusercontent.com/194784/76239295-9de26180-6231-11ea-98d2-58e6b4cad62e.png)

**Comment by NeOMakinG** [comment]
As I seen while working on this, it looks like it's unfixable without a big refacto on responsive if we want the scroll bar as the scrollbar require an overflow and so submenus should be outside the wrapper that needs the scroll bar... that's pretty hard

**Comment by colinegin** [comment]
Related to this fix : Fix BO menu scrollbar size on small screens #14689

**Comment by Progi1984** [comment]
Fixed by #18102
